css/revops.css

```diff
@@ -69,6 +69,44 @@ p {
   display:  flex;
   justify-content: space-evenly;
 }
+.revops-section__container--grid {
+  max-width: 1200px;
+  padding: 0 2rem 4rem;
+  text-align: left;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  padding-top: 1rem;
+  padding-bottom: 1rem;
```

Comment:
Multiple padding styles here. Choose if you want to use the `padding` or `padding-direction` apprach here instead of having both.

Reply:
My mistake, fixed 😊